### ISSUE TITLE
show text in the middle

### ISSUE THREAD

**Issue by luisrudge** [kimmobrunfeldt/progressbar.js#30]
can we add an api to insert text inside `ProgressBar.Circle`?
I mean, I wan't to add the percentage inside of it without having to rely on external html+css.
Thanks!

**Comment by kimmobrunfeldt** [comment]
I'm currently working on this https://github.com/kimmobrunfeldt/progressbar.js/pull/38

**Comment by kimmobrunfeldt** [comment]
There's now support for text. I released [0.7.0](https://github.com/kimmobrunfeldt/progressbar.js/releases/tag/0.7.0).
Example here: http://kimmobrunfeldt.github.io/progressbar.js/#example-circle-percent

Documentation: https://github.com/kimmobrunfeldt/progressbar.js/tree/0.7.0#shapecontainer-options

Quote from docs:
```json
   // Text options. Text element is a <p> element appended to container
    // You can add CSS rules for the text element with the className
    // NOTE: When text is set, 'position: relative' will be set to the
    // container for centering. You can also prevent all style modifications
    // with 'autoStyle: false'
    // Default: null
    text: {
        // Initial value for text.
        // Default: null
        value: 'Text',

        // Text color.
        // Default: same as stroke color (options.color)
        color: '#f00',

        // Class name for text element.
        // Default: 'progressbar-text'
        className: 'progressbar__label',

        // If true, CSS is automatically set for container and text element.
        // If you want to modify all CSS your self, set this to false
        // Default: true
        autoStyle: true
    },
```

**Comment by chcg823** [comment]
text is not working!
